pkg/kubecost/allocation_test.go

```diff
@@ -856,7 +864,7 @@ func TestAllocationSet_AggregateBy(t *testing.T) {
 
 	// 1a AggregationProperties=(Cluster)
 	as = generateAllocationSet(start)
-	err = as.AggregateBy(Properties{ClusterProp: ""}, nil)
+	err = as.AggregateBy([]string{string(AllocationClusterProp)}, nil)
```

Comment:
Understand why this is the way it is, but I think it's also okay to just use `[]string{"cluster"}` here for readability. Or we can change the type of `AllocationClusterProp` to be `string`. What do you think?

Reply:
I do not see a problem with this especially since, all of these are going through parseProperty anyway